### DISCUSSION TITLE
Silk.NET/docs/opengl/c1/2-hello-quad

### DISCUSSION THREAD

**Discussion by giscus** [dotnet/Silk.NET#1663] [Webpage Comments]
# Silk.NET/docs/opengl/c1/2-hello-quad

Your one-stop-shop for high performance .NET graphics & compute. A .NET Foundation project.

https://dotnet.github.io/Silk.NET/docs/opengl/c1/2-hello-quad.html

**Comment by TotylePepita** [comment]
How would you draw text on the screen?

**Reply by Beyley** [reply]
Id reccomend a library like [FontStashSharp](https://github.com/FontStashSharp/FontStashSharp), that is what i personally use on all my projects

**Reply by TotylePepita** [reply]
thanks for the recommendation, I've been searching for something like this for a while now. :)

**Comment by smn-1** [comment]
I hope more tutorials can be made, these are great.

**Comment by pazka** [comment]
Great tutorial for the moment. Thank you very much !

I still struggle to see how the link between the vertices and indices is done ( when I tried following the indices' order into the vertices array, I ended up drawing overlapping triangles...) but at least the end result compiles and displays as advertised ! 

A small fix : you forgot to specify to add `_ebo` to the top of our static class 
Also I think there is a bad redirection in your "next step" links at the end of your pages, coming from [the rendered presentation](https://dotnet.github.io/Silk.NET/docs/opengl/c1/2-hello-quad/)

**Reply by pazka** [reply]
> The false tells it we don't want to normalize the values

Also interestingly, if I change this parameter to `true`, the result doesn't change so this sentence and the following one was confusing to me
